src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -933,39 +933,62 @@ def make_output_quant_parameters(
         )
 
     def cnp_round(
-        self, x: Union[numpy.ndarray, fhe.tracing.Tracer], calibrate_rounding: bool
+        self,
+        x: Union[numpy.ndarray, fhe.tracing.Tracer],
+        calibrate_rounding: bool,
+        key: Optional[str] = None,
```

Comment:
I don't think key is a good naming here, what is it representing ? even reading the docstring does not make it learer, I would prefer a more explicit name if possible !

Reply:
A single quantized op can now have multiple rounding operations. The key refers to a specific rounding operation in a quantized op. I can call it `rounding_key` or `rounding_operation_id` maybe. Not sure it helps much more. I can edit the docstring to make it clearer maybe.

Reply:
Changed the name and updated doc string

Reply:
many thanks !